app/controllers/products_controller.rb

```diff
@@ -1,4 +1,4 @@
-class ProductsController < ApplicationController
+class V1::ProductsController < ApplicationController
   load_and_authorize_resource
```

Comment:
We don't need V1::ProductsController here. Generally, we use the versioning strategy for API only.

Reply:
forgot to remove that part